### ISSUE TITLE
[Regression] newline-after-description false-positive for description starting on first line of comment

### ISSUE THREAD

**Issue by kevinoid** [gajus/eslint-plugin-jsdoc#668]
## Expected behavior

`jsdoc/newline-after-description` would not warn about descriptions which start on the first comment line (i.e. the line with `/**`) and have a newline after the description, as it behaved in 30.7.13 and before.

## Actual behavior

    index.js
      2:0  error  There must be a newline after the description of the JSDoc block  jsdoc/newline-after-description
    
    ✖ 1 problem (1 error, 0 warnings)
      1 error and 0 warnings potentially fixable with the `--fix` option.

## ESLint Config

```json
{
  "plugins": [
    "jsdoc"
  ],
  "rules": {
    "jsdoc/newline-after-description": "error"
  }
}
```

## ESLint sample

```js
/** An example function.
 *
 * @returns {number} An example number.
 */
function example() {
  return 42;
}
```

## Environment

- Node version: v12.19.0
- ESLint version: v7.17.0
- `eslint-plugin-jsdoc` version: 31.0.2

Thanks,
Kevin

**Comment by kevinoid** [comment]
Lightning-fast as always.  Thanks again @brettz9!
